### PR TITLE
Add idmapped mounts for tmpfs

### PR DESCRIPTION
**Pull request by brauner**
Signed-off-by: Christian Brauner (Microsoft) <brauner@kernel.org>

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -510,3 +510,22 @@ associated problem space.
...
+  individual files or directories to expose information to containers/pods.
+  Instead of having to change ownership permanently allow them to use an
+  idmapped mount instead.
+
+  @rata and @giuseppe brought this suggestion forward. For Kubernetes it is
+  sufficient to support idmapped mounts of `tmpfs` instances mounted in the
+  initial user namespace. However, in the future idmapped
+  mounts of `tmpfs` instances mounted in user namespaces should be supported.
+  Other container runtimes want to make use of this. The kernel is able to
+  supprot this since at least `5.17`.
```
Comment (DaanDeMeyer) [inline README.md:526]: supprot => support

**Review by DaanDeMeyer — APPROVED** [review]

**Review by giuseppe — APPROVED** [review]
LGTM

**Review by rata — APPROVED** [review]
LGTM!

File: README.md [hunk 2]
```diff
@@ -510,3 +510,22 @@ associated problem space.
...
   RootImage= and friends in the systemd user manager.
+
+* Support idmapped mounts for tmpfs
+
+  **Use-Case:** Runtimes such as Kubernetes use a lot of `tmpfs` mounts of
+  individual files or directories to expose information to containers/pods.
+  Instead of having to change ownership permanently allow them to use an
+  idmapped mount instead.
+
+  @rata and @giuseppe brought this suggestion forward. For Kubernetes it is
```
Comment (t-lo) [inline README.md:521]:
```suggestion
  [@rata](https://github.com/rata/) and [@giuseppe](https://github.com/giuseppe/) brought this suggestion forward. For Kubernetes it is
```

**Pull request merged**